### ISSUE TITLE
Why are we not using docker-compose for development?

### ISSUE THREAD

**Issue by vinaypuppal** [coderplex-org/coderplex-backend-old#24]
* [x] I have searched the [issues](https://github.com/coderplex/coderplex/issues) of this repository and believe that this is not a duplicate.

# Expected Behavior
- Since we using docker-compose in production why don't we use it in development?
- It can remove the present [running-locally](https://github.com/coderplex/coderplex-backend/blob/develop/.github/CONTRIBUTING.md#running-locally) setup we have and we can extract all this logic to `Dockerfile` 
- And it has some benefits and drawbacks listed here https://runnable.com/blog/a-better-dev-workflow-with-docker-compose
- **And also I noticed present `CONTRIBUTING.md` files just lists `docker build` and `docker run`** I think they need to be replaced with `docker-compose up` so `postgres` is installed in production server. //cc @pbteja1998 

# Current Behavior
- Current setup is long and can be simplified with docker-compose.

**Comment by pbteja1998** [comment]
- I don't know much about `Docker` . So I can't tell why we are not using `Docker` for local development. In previous `README` written by @rrmerugu , we were not using `Docker` for local development. Hence I did not write about `Docker` in local development setup.
- We can simply write a `bash` script to simplify the [running-locally](https://github.com/coderplex/coderplex-backend/blob/develop/.github/CONTRIBUTING.md#running-locally) step, there is no need for the `Docker` just for the sake of simplifying the local setup process.
- Regarding last point, I just copied whatever @rrmerugu has written in `README` previously for `production` environment and copied that to `CONTRIBUTING.md`. 

So @rrmerugu can better answer this question.
